ui/app/components/app/menu-droppo.js

```diff
@@ -1,33 +1,92 @@
+import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { inherits } from 'util'
 import { findDOMNode } from 'react-dom'
 import ReactCSSTransitionGroup from 'react-transition-group/CSSTransitionGroup'
 
-export default MenuDroppoComponent
+export default class MenuDroppoComponent extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    innerStyle: PropTypes.object,
+    children: PropTypes.node.isRequired,
+    onClickOutside: PropTypes.func.isRequired,
+    containerClassName: PropTypes.string,
+    zIndex: PropTypes.number,
+    style: PropTypes.object.isRequired,
+    useCssTransition: PropTypes.bool.isRequired,
```

Comment:
```suggestion
    useCssTransition: PropTypes.bool,
```

This should not be required - this prop is not always provided. It's passed through via the `dropdown` component, but that component is not passed this prop in at least one place (`ui/app/components/app/dropdowns/network-dropdown.js`)